vita3k/crypto/include/crypto/aes.h

```diff
@@ -0,0 +1,179 @@
+#pragma once
```

Comment:
i took this from the rpcs3 code base, sorry i don't know the original repo

Reply:
No problem.

Reply:
It says in header 🙃 

https://github.com/cuberite/polarssl

Reply:
Here is the up to date repo that ARM is maintaining. https://github.com/ARMmbed/mbedtls
I don't expect you to use this (although it would be nice)- structure is totally different at first glance.

Reply:
oops i didn't pay attention to the header lol.
i'm sorry but i don't really wanna fiddle with this right now, maybe in a future PR i'll remove the individual files and use the repo instead.

Reply:
No problem :D